openfisca_uk/variables/revenue_scotland/lbtt.py

```diff
@@ -0,0 +1,113 @@
+from openfisca_uk.model_api import *
+
+
+class lbtt_on_residential_property(Variable):
```

Comment:
so i prefer lowercase acronyms but we should probably be consistent for now and switch them all over if we decide to do that